egs/wsj/s5/steps/nnet3/tdnn/make_configs.py

```diff
@@ -4,11 +4,14 @@
 from __future__ import print_function
 import os
 import argparse
+import shlex
 import sys
 import warnings
 import copy
 import imp
 import ast
+import scipy.signal as signal
```

Comment:
Yes i resolve the conflicts in make_configs.py .
But i don't know if i should force add_lda to false when cnn is used.